builds/e2e/nested-connectivity.yaml

```diff
@@ -169,15 +172,16 @@ stages:
           testInfo.testName: '$(testInfo.testName)'
   - job:  Clean_up      
     dependsOn:
+      - Lock_Nested_Agents_mqtt
       - SetupVM_level5_mqtt
       - SetupVM_level4_mqtt
       - Deploy_connectivity_Linux_Amd64
```

Comment:
Actually, don't we still want them to clean up despite the success/fail status? 
It should always be cleaning up. This might be a bug?

Reply:
I was wondering the same thing. But there should be an `always()` somewhere around here. Dependencies are just for referencing the output variables.